packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -106,6 +119,14 @@ describe('CheckoutModule', () => {
         expect(state).toEqual(defaultState);
     });
 
+    describe('getters ::', () => {
+        describe('analyticsData` ::', () => {
+            it('should return `isLoggedIn', () => {
+                expect(analyticsData(state)).toMatchSnapshot();
```

Comment:
I think the test description is a little bit misleading for this check as you aren't checking specifically for the presence of `isLoggedIn` (someone could update the test without updating the description and it would still pass) – you're just checking for a valid snapshot.

I'd either change the name of the test to reference the snapshot, or try and check more specifically for `isLoggedIn` being present in `analyticsData(state)`

Reply:
This, I think, was a half-written test, sorry. This has now been removed